docs/tools/platformsh.md

```diff
@@ -0,0 +1,27 @@
+# Platform.sh
+
+Platform.sh offers a command line tool to help . It enables you to do almost everything in a terminal that you can do in the Dashboard, and much more.
+
+Platform.sh's CLI is pre-installed in the `cli` container. Note: required `docksal/cli` version: 2.3.0+.
+
+
+## Configuration
+
+Follow the [official docs](https://docs.platform.sh/gettingstarted/cli/api-tokens.html) to generate your API token.
+
+Add the token to `$HOME/docksal/docksal.env`:
```

Comment:
Rewrite to use `fin config set --global`. May still mention that the variable will be stored in `$HOME/docksal/docksal.env`